components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -352,8 +352,9 @@ export function CreateWorkspacePage() {
         <div className="flex flex-col mt-32 mx-auto ">
             <div className="flex flex-col max-h-screen max-w-lg mx-auto items-center w-full">
                 <Heading1>New Workspace</Heading1>
-                <div className="text-gray-500 text-center text-base">
-                    Start a new workspace with the following options.
+                <div className="text-gray-500 text-center text-base mx-6">
+                    Start a workspace with the following options in organization{" "}
+                    <span className="font-semibold">{currentOrg?.name}</span>.
```

Comment:
**suggestion**: Minor restructure[^1][^2] of the subheading. Nice catch[[1](https://github.com/gitpod-io/gitpod/pull/17701#issuecomment-1557369345)], @easyCZ! ⚾ 

<img width="1318" alt="Frame 1471" src="https://github.com/gitpod-io/gitpod/assets/120486/63ce585c-eff4-4d98-9bee-f4fc9170f0ce">

[^1]: Using a new _Create_ verb for the first time. Alternatively, we could stick with _Start_.
[^2]: Dropping the options reference which is repeated below the _Continue_ action.